src/styles/global.css

```
@@ -33,10 +34,24 @@
+      ui-sans-serif,
+      system-ui,
+      -apple-system,
+      BlinkMacSystemFont,
+      Segoe UI,
+      Roboto,
+      Helvetica Neue,
+      Arial,
+      Noto Sans,
+      sans-serif,
+      Apple Color Emoji,
+      Segoe UI Emoji,
```

Biome (1.9.4)

[error] Duplicate font names are redundant and unnecessary: Segoe

Remove duplicate font names within the property

(lint/suspicious/noDuplicateFontNames)